packages/core/resolve-testing-tools/src/aggregate-assertions.ts

```diff
@@ -19,6 +19,19 @@ type BDDAggregateAssertionContext = {
   [symbol]: BDDAggregateAssertionState
 }
 
+const stringifyError = (error: any): string => {
```

Comment:
Does such function *really* considered? Which use cases have forced such counter-intuitive implementation?

| Input | Output |
|-------|--------|
| 0     | "no error"      |
| 1     | "1"      |
| false     | "no error"      |
| true     | "true"      |
| ""     | "no error"      |
| "abc"    | "abc"      |
| new String("abc")    | "\\"abc\\""      |
| String("abc")    | "abc"      |
| new Error()    | "\\"\\""      |
| new Error("abc")    | "abc"      |

Reply:
```js
const stringifyError = (error: any): string => {
  if (error == null) {
    return 'no error'
  }
  return error.toString()
}
```
Maybe like this?